spec/requests/organizer_can_set_group_category_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+describe 'GET /cagegories/:id/groups' do
+  let!(:category_1) { create(:category) } 
+  let!(:category_2) { create(:category) } 
+
```

Comment:
Shouldn't it be just 'GET /cagegories/:id'?

Reply:
These have both been fixed.